Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md

```diff
@@ -208,8 +208,20 @@ You MUST include an estimate of all the embodied emissions for the hardware used
 You MAY use simple models to estimate embodied emissions; however, you SHOULD use the most granular data possible and ideally emissions data from a devices life cycle analysis when calculating your embodied carbon.
 
 
-### Preset List for Baselines
-[placeholder]
+### Preset List for Baselines (R)
+Acceptable baselines include:
+- API call/request 
+- Benchmark 
+- User
+- Machine
+- Minute/time unit
+- Device
```

Comment:
Does this (and other similar items like physical site, or data volume) need some form of sizing? I can clearly see how User, or API call relates to R. But physical site, data volume, and device, seem like they could have significant variance.

Reply:
- I think size is fine, like GB, MB? Or are you saying we need to list every term that can be used to define the size of a data?

- Device I think makes sense if you are perhaps an IoT cloud service and you scale by the number of devices that are sending you data?

Reply:
We should definitely have some sizing suggestions, and the fact that we if reword it as "suggested" as suggested (pun intended! ;) ) by @jawache then that would make it clearer.